src/yaourt.sh.in

```diff
@@ -127,6 +128,15 @@ yaourt_install_packages() {
 	prepare_status_list
 	SP_ARG="" sync_packages "${ARGS[@]}" || return $?
 	analyse_status_list
+
+	# Trigger Backup
+	rnum=${#ARGS[@]}
+	fnum=${#ERROR_PKGS[@]}
+	if [[ "$((rnum - fnum))" -gt "0" ]]; then
+		load_lib gist_backup
+		gist_backup_poll
```

Comment:
I disagree:

    $ yaourt -S tree
    resolving dependencies...
    looking for conflicting packages...

    Package (1)  New Version  Net Change

    extra/tree   1.7.0-1        0.08 MiB

    Total Installed Size:  0.08 MiB

    :: Proceed with installation? [Y/n] 
    (1/1) checking keys in keyring [#################################################] 100%
    ....
    (1/1) installing tree          [#################################################] 100%

    Gist Backup is a feature which saves an updated
    version of your package list privately to your GitHub Account
    with every package installation.


    ==> Do you wish to install Gist Backup ? [y/N]
    ==> ------------------------------------------
    ==>